### PR TITLE
fix(chat): fix quick reaction context menu

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- slight store cleanup, remove unused state, remove unused action
- fix quick reaction in message context menu

**Which issue(s) this PR fixes** 🔨
AP-1085

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: store/ui/actions.ts [hunk 1]
```diff
@@ -30,10 +30,6 @@ export default {
   setActiveChannel({ commit }: ActionsArguments<UIState>, channel: Channel) {
     commit('setActiveChannel', channel)
   },
-  addReaction({ commit }: ActionsArguments<UIState>, reaction: any) {
-    commit('addReaction', reaction)
-    commit('updateRecentReactions', reaction.emoji)
```
Comment (molimauro) [inline store/ui/actions.ts:35]: wo this recent reactions arent updated anymore right?

Comment (josephmcg) [reply store/ui/actions.ts:35]: yes, but I believe that was unused state from when the app was being mocked. the actual data was being tracked in `mostEmojiUsed`, which could probably be renamed to something more natural

Comment (molimauro) [reply store/ui/actions.ts:35]: ok ty!! :)

**Review by molimauro — APPROVED** [review]

File: store/ui/actions.ts (hunk 1 above)
Comment (josephmcg) [reply store/ui/actions.ts:35]: I guess those associated mutations could be removed too, didn't notice that before

**Comment by josephmcg** [comment]
accidentally added some unrelated enum file lol.. fixing

**Comment by molimauro** [comment]
thanks!!

**Review by stavares843 — APPROVED** [review]

**Pull request merged**